articles/libraries/auth0-swift/index.md

```diff
@@ -77,7 +79,30 @@ Auth0
    }
 ```
 
-#### Passwordless Login
+### Passwordless Login
+
+Logging in with a Passwordless is slightly different. Passwordless can be done via email or via SMS, and either by sending the user a code, or sending them a link which contains a code. 
+
+#### Passwordless Parameters
+
+As you can see, Passwordless authentication can be started with a variety of different parameters. 
+
+```
+.startPasswordless(email/phoneNumber: String, type: String, connection: String)
+```
+
+- **Parameter One** - Either the parameter `email` or `phoneNumber`, depending on which you intend to use. The value of either should be a string.
+- **Parameter Two** - The parameter `type`, its value should be either `.Code` or `.iOSLink`. The default is `.Code` (a code is sent to the user, then they input it in a secondary screen), but if you have iOS Universal Links configured, you can use `.iOSLink`.
+- **Parameter Three** - The parameter `connection`, its value should be the name of the connection, defaults to `sms`.
+
+
+#### How Passwordless Works
+
+Passwordless requires two steps, at the heart of things. Requesting the code, and inputting the code. When using links, this is slightly different, because the user does not have to input a code themselves - but the code is just included in the URL.
+
+**Step 1:** Request the code
+
+In this example, requesting the code is done by calling `startPasswordless` with the user's email, and the type of connection. The `type` parameter will default to `Code`. On success, you'll probably display a notice to the user that their code is on the way, and perhaps route them to a view to input that code.
```

Comment:
Use the appropriate method example,  also `you'll probably display a notice` should be worded as a recommendation of best practice.